examples/php-xhr/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask
+werkzeug
+flask-cors
```

Comment:
probably an extraneous file? python-xhr and php-xhr both have a requirements.txt, should only be python i think?

Reply:
Yes, thanks for noting this.